### ISSUE TITLE
Document how to customize BETYdb

### ISSUE THREAD

**Issue by dlebauer** [PecanProject/betydb-documentation#12]
Can start by copy-pasting markdown from https://github.com/PecanProject/bety/pull/447

**Comment by KristinaRiemer** [comment]
@dlebauer do you want to assign this to me?

**Comment by dlebauer** [comment]
@KristinaRiemer yes, please coordinate with @gsrohde about which branch to use for new content. I think he has been migrating from gitbook to bookdown but am not sure of the status
